starlette/py.typed

```diff
@@ -0,0 +1 @@
+
```

Comment:
just a nitpick: doesn't need an empty line, does it?

Reply:
Whatever makes you happy